pkg/model/metric.go

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) 2017 Huawei Technologies Co., Ltd. All Rights Reserved.
```

Comment:
Please change the copyright time to 2019

Reply:
@leonwanghui , Fixed. Thanks